src/utils/getCustomConfiguration.ts

```diff
@@ -6,12 +6,14 @@ import packageJSON from '../../package.json';
 import { logger } from './debug';
 
 const debug = logger('customConfiguration');
+
 export type CustomConfig = {
   includes?: string[];
+  output?: string;
```

Comment:
Re-ordering in some places so the options are in the same order as in the documentation for consistency and readability.